src/index.ts

```diff
@@ -95,11 +105,56 @@ function gameSelectCard(card: Card): void {
 }
 
 function isSet(cards: Set<Card>): boolean {
-    return true;
+    // mini function that checks if that an attribute is valid, i.e. all the same, or all different
+    function validAttribute(levels: Array<number>){
+        if ((levels[0] == levels[1] && levels[1] == levels[2])||(levels[0] != levels[1] && levels[0] != levels[2] && levels[1] != levels[2])){
+            return true
+        }
+        else{
+            return false
+        }
```

Comment:
```suggestion
        return ((levels[0] == levels[1] && levels[1] == levels[2])||(levels[0] != levels[1] && levels[0] != levels[2] && levels[1] != levels[2]))
```

you accidentally wrote something in the form `if (true) {return true} else {return false}`, so just return it directly.